specification/openai/resource-manager/Microsoft.OpenAI/preview/2021-09-01-preview/openai.json

```diff
@@ -0,0 +1,1990 @@
+      "get": {
+        "tags": [
+          "OpenAIAccounts"
+        ],
+        "description": "Get usages for the requested OpenAI account",
+        "x-ms-examples": {
+          "Get Usages": {
+            "$ref": "./examples/GetUsages.json"
+          }
+        },
+        "operationId": "Accounts_ListUsages",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/usageFilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The usages for OpenAI account were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/UsageListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource does not exist, 404 (NotFound) will be returned.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.OpenAI/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all the available OpenAI account operations.",
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "Get Operations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.OpenAI/locations/{location}/checkSkuAvailability": {
```

Comment:
>checkSkuAvailability [](http://example.com/codeflow?start=85&length=20)

As commented above, can we define location based list API? Rather then POST and follow existing pattern.

Reply:
The difference between checkSkuAvailability and get SKUS API, is checkSkuAvailability is implemented by RP which has resource quota check, but get SKUs API is from ARM, so it don't check the resource limit. We can probably remove checkSkuAvailability  from OpenAI since it's not that important anymore with the support of preflight.